pkg/acceptance/terraform/azure/variables.tf

```diff
@@ -60,11 +60,14 @@ variable "azure_vhd_storage_account" {
   # Using a static storage account name also has the benefit of making it
   # easier to expunge leaked VHDs, which continue to occur as of 2017-08-09.
   # Only lowercase letters and numbers are allowed by Azure.
-  default = "cockroachnightlyvhd"
+  #
+  # N.B. Keep this in sync with build/teamcity-reset-nightlies.sh.
+  default = "cockroachnightlyeastvhd"
```

Comment:
This says `east` and backup_test.go below is `west`. Is that correct?

Reply:
Yup, exactly. eastus doesn't support Ls-series VMs 🤦‍♂️, which we need for the backup/restore tests.